### DISCUSSION TITLE
Unable to login and reset password

### DISCUSSION THREAD

**Discussion by esskay303** [trobfab/fabrik#615] [General]
Hello, I am unable to login at fabrikar.com forum. I have tried to reset password, username but could not succeed. I never got any email. I also tried to create another account with a new email but I am unable to create a new account too. The system is not sending any email. Can you please look into it?

**Comment by trob** [comment]
Fabrikar.com account emails stopped long ago.
I can't help, it's not our site, I have only some forum privileges.

I think it's still possible to create a new account for the forum, is it?

**Comment by esskay303** [comment]
Hi. Thanks for your reply. I did try to create a new account several times but I keep getting error which says 'Notice
Could not execute: /usr/sbin/sendmail
Error
An error was encountered while sending the registration email. A message has been sent to the administrator of this site.'

Please see the attached screenshot.
![Fabrik](https://github.com/trobfab/fabrik/assets/4937306/d6621ae6-1437-4ebb-9041-062bc6d458b6)

**Comment by trob** [comment]
Yes, as I said emails don't do since long time.
But did you try to log in anyway?

**Comment by esskay303** [comment]
I did try but there was an error. See the screenshot. However, when I gtried to login directly to the Forum section I was able to login. Thanks.
![Fabrik2](https://github.com/trobfab/fabrik/assets/4937306/eebd0a69-301b-47b0-ba17-a5e6a48710e1)
